src/www/ui/api/index.php

```diff
@@ -269,6 +271,13 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
     $app->any('/{params:.*}', BadRequestController::class);
   });
 
+/////////////////////////FILE INFO////////////////////
+$app->group('/file',
+  function (\Slim\Routing\RouteCollectorProxy $app) {
+    $app->get('/upload/{id:\\d+}/item/{itemId:\\d+}/info', FileInfoController::class . ':getItemInfo');
+    $app->any('/{params:.*}', BadRequestController::class);
+  });
+
```

Comment:
Since the endpoint is move to `/uploads/{id}/item/{itemId}/info`, shouldn't this be removed as well?